frontend/apps/reader/skimtowidget.lua

```diff
@@ -134,10 +134,7 @@ function SkimToWidget:init()
         width = self.button_width,
         show_parent = self,
         callback = function()
-            self.curr_page = self.curr_page - 1
-            self:update()
-            self:addOriginToLocationStack()
-            self.ui:handleEvent(Event:new("GotoPage", self.curr_page))
+            self:gotoPageCallback(self.curr_page - 1)
```

Comment:
Is there a maintainability rationale behind including the current use case for the function in the name?
```suggestion
            self:goToPage(self.curr_page - 1)
```

Reply:
it was originally less confusing to me vs the goToPage event, should i change it back to `goToPage`?

Reply:
done